apps/web/src/pages/Teams/ViewTeamPage/ViewTeamPage.tsx

```diff
@@ -1,12 +1,77 @@
 import * as React from "react";
 import { useIntl } from "react-intl";
 import { BuildingOffice2Icon } from "@heroicons/react/24/outline";
-
+import { Scalars, useViewTeamQuery } from "~/api/generated";
+import { useParams } from "react-router-dom";
 import PageHeader from "@common/components/PageHeader";
 import SEO from "@common/components/SEO/SEO";
+import { commonMessages } from "@common/messages";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+import Link from "@common/components/Link";
+import useRoutes from "~/hooks/useRoutes";
+import Heading from "@common/components/Heading";
+import { getLocalizedName } from "@common/helpers/localize";
+import { ViewTeam } from "./components/ViewTeam";
+
+type RouteParams = {
+  teamId: Scalars["ID"];
+};
+
+interface ViewTeamApiProps {
+  pageTitle: string;
+}
+
+export const ViewTeamApi = ({ pageTitle }: ViewTeamApiProps) => {
+  const intl = useIntl();
+  const { teamId } = useParams<RouteParams>();
+  const [{ data, fetching, error }] = useViewTeamQuery({
+    variables: { id: teamId || "" },
+  });
+
+  return (
+    <Pending fetching={fetching} error={error}>
+      {data?.team ? (
+        <>
+          <PageHeader icon={BuildingOffice2Icon}>{pageTitle}</PageHeader>
```

Comment:
Thoughts on starting to alias icons, instead of using stock names? Like if `BuildingOffice2Icon` is our `TeamIcon`, it could be nice to just wrap the icon in our repo, and create that clarity in our chosen set of components :)